src/renderer/types/global.d.ts

```diff
@@ -1 +1,3 @@
 declare const __static: string;
+// Bug fix for ts error in react-use
```

Comment:
Can you be more specific, which hook caused this error? So that if that hook is ever deprecated, we know when to remove this.